hera_sim/visibilities/healvis_wrapper.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import division
+
+import healpy
+import numpy as np
+from cached_property import cached_property
+
+import pyuvsim
+
+from .simulators import VisibilitySimulator
+from astropy import constants as cnst
+
+try:
+    from healvis.beam_model import AnalyticBeam
+    from healvis.simulator import setup_observatory_from_uvdata
+    from healvis.sky_model import SkyModel
+except ImportError:
+    raise ImportError("to use the healvis wrapper, you must install healvis!")
+
+
+class HealVis(VisibilitySimulator):
+    """Wrapper for healvis to produce visibilities from HEALPix maps."""
+
+    point_source_ability = False
+
+    def __init__(self, fov=180, nprocesses=1, sky_ref_chan=0, **kwargs):
+        """
+        Parameters
+        ----------
+        fov : float
+            Field of view (diameter) in degrees. Defaults to 180.
+        nprocesses : int
+            Number of concurrent processes. Defaults to 1.
+        sky_ref_chan : float
+            Frequency reference channel. Defaults to 0.
+        **kwargs
+            Arguments of :class:`VisibilitySimulator`.
+        """
+        self.fov = fov
+        self._nprocs = nprocesses
+        self._sky_ref_chan = sky_ref_chan
+
+        # A bit of a hack here because healvis uses its own AnalyticBeam, and
+        # doesn't check if you are using pyuvsim's one. This should be fixed.
```

Comment:
Can you make an issue noting that this should be fixed?

Reply:
I think this really needs to be fixed on `healvis` rather than here. I think @aelanman is already on it.